packages/components/message/src/message-method.ts

```diff
@@ -16,6 +17,9 @@ let seed = 1
 
 const message: MessageFn & Partial<Message> = function (options = {}) {
   if (!isClient) return { close: () => undefined }
+  if (messageConfig.max && instances.length >= messageConfig.max) {
```

Comment:
We want to make sure that the value passed to `messageConfig.max` is a number.

Do:

```ts
isNumber(messageConfig.max) && instances.length >= messageConfig.max
```

Reply:
Done.